app/support/reconcilable_account.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 module ReconcilableAccount
-
+  def reconcilable?
```

Comment:
I think it's worth adding a comment here to make it more obvious that this is a class method